stbt/__init__.py

```diff
@@ -140,6 +141,31 @@ def match(image, frame=None, match_parameters=None, region=Region.ALL):
     return _dut.match(image, frame, match_parameters, region)
 
 
+def match_all(image, frame=None, match_parameters=None, region=Region.ALL):
+    """
+    Search for all instances of an image in a single video frame.
+
+    Arguments are the same as `match`.
+
+    :returns:
+      An iterator of zero or more `MatchResult` objects (one for each position
+      in the frame where ``image`` matches).
+
+    Examples:
+
+    .. code-block:: python
+
+        all_buttons = list(stbt.match_all("button.png"))
+
+        for match_result in stbt.match_all("button.png"):
+            # do something with match_result here
+            ...
```

Comment:
Perhaps this should be two seperate code-blocks to make it clear that these lines of code are unrelated.